README.md

```diff
@@ -110,7 +110,7 @@ Retrieves only the metadata from an object in an Amazon S3 bucket. Uses [s3:head
 #### s3.deleteObject({ bucket: String, key: String })
 Delete object from an Amazon S3 bucket at key. Uses [s3:deleteObject](https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#deleteObject-property).
 
-#### s3.listObjects({ bucket: String, limit: Number, startAfter: String, prefix: String })
```

Comment:
Should we put in a description for `stopAfter`?
`stopAfter: used to stop paging if the first entry in the key is larger than this parameter.`?

Reply:
Sure 

```
stopAfter: If provided paging is stopped if the last key returned is larger or equal to this parameter. Only appropriate keys are then returned
```